migfinder/migfinder.py

```diff
@@ -869,22 +899,25 @@ def posproc2(out,k_orf = 0, d_CDS_attC = 500, dist_threshold=4000):
 							break
 			fcoord.close()
 
-			os.system("rm orfresults/"+out+"_ALLorf.faa")
-			os.system("rm orfresults/"+out+"_ALLorf.fna")
-		# ------------------------------------------------------ #
+			os.unlink(output_file_faa)
+			os.unlink(output_file_fna)
+
+	# ------------------------------------------------------ #
 	# ---------------------------------------------------- #
 	# Saving info about removed ORFs
-		fileout = open(out+".filtering",'a')
-		fileout.write("Del_attC_ol:	"+str(remove_overlap_attC)+'\n')
-		fileout.write("Total_orfs:	"+str(total_orf)+'\n')
-		fileout.write("Del_orf_k:		"+str(remove_k)+'\n')
-		fileout.write("Del_orf_ol:	"+str(remove_overlap)+'\n')
-		fileout.write("Del_orf_ni:	"+str(remove_not_int)+'\n')
-		#fileout.write("Del_attC_ws:	"+str(remove_attC_wrongstrand)+'\n')
-		fileout.write("\n")
-		fileout.write("ol= overlap, k= score_below_threshold, ni= not_integron\n")
-		fileout.write("ws= wrong_strand\n")
-		fileout.close()
+
+		with open(f"{output_dir}/{output_file}.filtering", 'a') as fileout:	
+			fileout = open(output_file+".filtering",'a')
```

Comment:
2 comments, this line is opening `f"{output_dir}/{output_file}.filtering"`
This one is a tsv file, so it's better to use the python csv module to do this:

```python
import csv
...
with open(f"{output_dir}/{output_file}.filtering", 'a') as fileout:
    writer = csv.writer(fileout, delimiter="\t")
    writer.writerow(["Del_attC_ol:", str(remove_overlap_attC)])
    writer.writerow(["Total_orfs:", str(total_orf)])
    ...
    fileout.write("ol= overlap, k= score_below_threshold, ni= not_integron", )
    fileout.write("ws= wrong_strand\n")

```

Reply:
Ok, I updated the code replacing the suggested section.